assets/sass/_mixins.scss

```diff
@@ -98,7 +98,6 @@
         // eg. 80% of the max
         width: #{$widestScreen / 100 * 80};
         @if not $skipMargins {
-            height: 100%;
```

Comment:
Noticed this was causing some display bugs with overflowing content when this class is used. Removing it fixes those bugs and afaict doesn't cause any regressions elsewhere.